web_autoflip_demo/autoflip_demo_single_worker/autoflip.html

```diff
@@ -0,0 +1,64 @@
+<!-- 
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. 
+-->
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Video File upload and play demo</title>
+  </head>
+  <body>
+    <div>
+      <h1>Upload a video to test!</h1>
+      <input type="file" id="video-upload" accept="video/*" />
+      <button id="start-worker">Start Worker</button>
+      <h1>Preview</h1>
+      <div>
+        <video id="video-preview" width="400" height="200" autoplay loop controls></video>
+      </div>
+      <h1>Autoflip</h1>
+      <div>
+        <video id="video-display" width="400" height="200" autoplay loop controls></video>
+      </div>
+      <svg id="mask" style="position: absolute; left: 8; top: 465; width: 400; height: 200; pointer-events: none;">
```

Comment:
try to keep the html elements from going beyond ~80 characters. Like you did below put the separate attributes into new lines

Reply:
Fixed.